lmdeploy/turbomind/turbomind.py

```diff
@@ -640,6 +620,12 @@ def _get_generation_config(self, cfg: GenerationConfig):
         c.top_p = cfg.top_p
         c.min_p = cfg.min_p
         c.temperature = cfg.temperature
+        if cfg.stop_token_ids:
```

Comment:
LMDeploy's GenerationConfig defines stop_token_id, bad_token_id
But turbomind's GenerationConfig defines eos_id, stop_id, and bad_id.
Here we map stop_token_id to both eos_id and stop_id. It is confusing.

Reply:
We need to discuss about it